spec/spec_helper.rb

```diff
@@ -69,6 +69,10 @@
   # rspec-rails.
   config.infer_base_class_for_anonymous_controllers = false
 
+  # TODO - remove this line after Spree upgrade. Current Spree specs do not have `type: :controller` definitions.
+  # This is a workaround to pass tests in Travis i.e. https://travis-ci.org/openfoodfoundation/openfoodnetwork/jobs/235601640
+  config.infer_spec_type_from_file_location!
+
```

Comment:
FYI @jeronimo, you can just add `type: :controller` to `spec/controllers/line_items_controller_spec.rb` and it will be fine, no need to use `infer_spec_type_from_file_location`. :)